File: Data Structures & Algorithms/anagram-groups/submission-0.go
func groupAnagrams(strs []string) [][]string {
    buckets := make(map[[26]int][]string)
    idx := func (char byte) int {
        return int(char - 'a')
    }
    for _, str := range strs {
        var bucket [26]int
        for i := range str {
            bucket[idx(str[i])]++
        }
        buckets[bucket] = append(buckets[bucket], str)
    }
    res := make([][]string, 0, len(buckets))
    for _, group := range buckets {
        res = append(res, group)
    }
    return res
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        n = len(nums)
        res = 0
        l = 0
        for r in range(n):
            if nums[r] == 1:
                if r-l+1 > res:
                    res = r-l+1
            else:
                l = r+1
        return res